development-defaults.ini

```diff
@@ -165,9 +165,7 @@ stripe_error = "Stripe Error Override"
 
 [[door_payment_method]]
 cash   = "Pay with cash"
-stripe = "Pay with credit card now (faster)"
-manual = "Pay with credit card at the registration desk (slower)"
-group  = "Taking an unassigned Group badge (group leader must be present)"
+manual = "Pay with credit card at the registration desk"
```

Comment:
These should probably still permanently live here because otherwise we could get an error when trying to save an existing Attendee who has already been assigned one of these values.